packages/material-ui/src/ListItem/ListItem.js

```diff
@@ -145,13 +145,15 @@ export const ListItemRoot = experimentalStyled(
   }),
 }));
 
+const containerOverridesResolver = (props, styles) => styles.root;
+
 const ListItemContainer = experimentalStyled(
   'li',
   {},
   {
-    name: 'MuiListItem',
-    slot: 'Container',
-    overridesResolver,
+    name: 'MuiListItemContainer',
```

Comment:
Te changes is done os that we can have a different key under `theme.components`, otherwise the styles for the `ListItem` would be applied on the container too.